src/components/profile/Profile.vue

```diff
@@ -506,9 +505,22 @@ export default {
   border-radius: var(--cardRadius);
   position: relative;
 }
+
+.profile__section:not(.profile__section--editing):not(.first) {
+  padding: calc(2px + 1.5em);
+  box-sizing: content-box;
```

Comment:
![image](https://user-images.githubusercontent.com/3604775/63025510-a0522800-bea9-11e9-96ab-6d3df2c36a47.png)
That's happening when we change the box-sizing model here. I didn't dig to figure out why. As mentioned, it was a choice to change to `border-box` globally. If we want to change this we should make this part of a bigger refactoring. As for now I suggest we keep it that `border-box` way and defer this.

Reply:
Good call. That was actually a style to remove and I hadn't seen that behaviour till you pointed it out. Thanks!